packages/components/src/public/QueryModel/ViewMenu.tsx

```diff
@@ -41,10 +40,11 @@ export class ViewMenu extends PureComponent<ViewMenuProps> {
         const noViews = publicViews.length === 0 && privateViews.length === 0;
         const _hideEmptyViewMenu = getQueryMetadata().get('hideEmptyViewMenu', hideEmptyViewMenu);
         const hidden = _hideEmptyViewMenu && noViews;
-        const disabled = isLoading || (noViews && !isCustomizeViewsInAppEnabled());
+        const disabled = isLoading || (noViews && !allowViewCustomization);
```

Comment:
I know allowViewCustomization is not introduced in this story, but I find it weird that hideEmptyViewMenu overrides allowViewCustomization. I would think if allowViewCustomization= true, then the manage portion of the view menu would still show when there are no views, even if hideEmptyViewMenu=true

Reply:
Yes, you're right.  Evolution of some flawed logic, now simplified.